app/assets/stylesheets/content/_attributes_key_value.sass

```diff
@@ -27,19 +27,21 @@
 //++
 
 .attributes-key-value
-  @include grid-block
-  @include grid-layout(2)
-  @include grid-visible-overflow
+  display: flex
+  flex-wrap: wrap
   font-size: 0.875rem
   line-height: 1.6
 
   .attributes-key-value--key
     @extend .form--label
-    @include grid-size(4)
-    @include text-shortener
     display: flex
+    // We have to find a middle way between
+    // (a) avoiding label wrap and
+    // (b) ugly content wrap (e.g. in the label)
+    // These values are what matches best at the moment. So be careful when changing them.
+    flex: 1 0 45%
```

Comment:
In my eyes is 45 too much for the general width. That looks a bit wasted on all other pages. Although I am not a fan of adding special rules I suggest to keep it the way it was before. So 45% on the WP view, and 33% as default.

Reply:
Done :)